tests/test_pygtracker.py

```diff
@@ -1,15 +1,93 @@
 from pygtracker import __version__
 from pygtracker import pygtracker
+from pytest import raises
+import pandas as pd
+import numpy as np
+from pandas._testing import assert_frame_equal
 
 def test_version():
     assert __version__ == '0.1.0'
     
 # Start tests for register_courses
 
+def test_register_courses_course_id_valid():
+    tracker=pygtracker.GradeTracker()
+    df=pd.DataFrame()
+    df['course_id']=[555]
```

Comment:
It is better to declare a separate helper function to generate the df frame that you are using here.

`def generate_df_register_courses(course_id, assessment_id, weight)`